keyboards/saevus/cor/readme.md

```diff
@@ -0,0 +1,26 @@
+# saevus cor
+
+![saevus cor](https://i.imgur.com/jqoNEcHl.jpeg)
```

Comment:
Is this meant to be an image of the empty case? 

Ideally, should show the PCB and/or assembled keyboard.

Reply:
Sorry about that. I've updated the image to an assembled board.